.travis.yml

```diff
@@ -2,26 +2,11 @@ dist: trusty
 language: elixir
 
 elixir:
-    - 1.2.6
-    - 1.3.4
-    - 1.4.1
-    - 1.5.1
     - 1.6.1
+    - 1.7.4
```

Comment:
Should we add 1.8 and 1.9 since that's what the PR is updating?

Reply:
Done, but it's about their CI system building multiple versions in parallel - https://travis-ci.org/xerions/phoenix_swagger/builds/561320905 - I'm not sure if there is any use for us.

Reply:
If we want the changes to go back upstream, then we should probably test that they're working with the new versions.